web/src/services/api/widgetsApi/utils.ts

```diff
@@ -40,8 +40,9 @@ export const getInstallableWidgets = (rawScene?: GetSceneQuery) => {
       return plugin?.extensions
         .filter(
           e =>
-            e.type === PluginExtensionType.Widget &&
-            AVAILABLE_WIDGET_IDS.includes(`reearth/${e.extensionId}`),
+            (e.type === PluginExtensionType.Widget &&
+              AVAILABLE_WIDGET_IDS.includes(`reearth/${e.extensionId}`)) ||
+            plugin.id !== "reearth",
```

Comment:
```
(e.type === PluginExtensionType.Widget &&
              (AVAILABLE_WIDGET_IDS.includes(`reearth/${e.extensionId}`)) ||
            plugin.id !== "reearth")
```

Reply:
This is the widgetsApi, so should be widget extension 👍🏻

Reply:
Updated!